image-bundle.yaml

```diff
@@ -1,7 +1,7 @@
 schema_version: 1
 name: "rhpam-7/rhpam-kogito-operator-bundle"
 description: "RHPAM Kogito Operator Bundle"
-version: "7.11"
+version: ""7.11.0"
```

Comment:
double quotes are added twice, here and on the next updates as well

Reply:
Sorry I ran something to mass update these, my bad, will fix.

Reply:
np :)